### PR TITLE
Adding google oauth to our navbar. 

### PR DESCRIPTION
**Pull request by nschickle**
We can do customization later but I wanted to get it set up first. Not sure how it took me hours to figure it out seeing how little I actually added but we're getting there...

### CODE REVIEW

File: credentials.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"web":{"client_id":"528310070004-u0clc84o9iktpqi6tjujqe9pq9f6ns2n.apps.googleusercontent.com","project_id":"tasktabs-1585076424102","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"-3M3V-BxaZB1mc74wmdh6SsJ","redirect_uris":["https://tasktabs-frontend-staging.herokuapp.com/"]}}
```
Comment (JustinLaGree) [inline credentials.json:1]: Can you pretty-print this? I find this very unreadable.

File: src/components/navBar.tsx [hunk 2]
```diff
@@ -9,14 +9,16 @@ import Dropdown from 'react-bootstrap/Dropdown';
...
-        height: 55,
+        height: 75,
         borderColor: "gray",
         borderStyle: "solid"
     },
     button: {
         fontSize: 24,
-        height: 50,
-        borderStyle: "none"
+        height: 48,
```
Comment (JustinLaGree) [inline src/components/navBar.tsx:18]: Make these styling changes on the g-signin2 class. This way all buttons won't be styled like this.

File: credentials.json (hunk 1 above)
Comment (nschickle) [reply credentials.json]: I fixed it a bit

File: src/components/navBar.tsx (hunk 2 above)
Comment (nschickle) [reply src/components/navBar.tsx]: Reversed this back to keep buttons the same and added to the g-signin2 class.

**Pull request merged**